packages/host/app/lib/current-run.ts

```diff
@@ -540,11 +540,17 @@ export class CurrentRun {
     let types: string[] = [];
     let fullRef: CodeRef = ref;
     while (fullRef) {
-      let loadedCard = await loadCard(fullRef, { loader: this.loader });
-      let loadedCardRef = identifyCard(loadedCard);
-      if (!loadedCardRef) {
-        throw new Error(`could not identify card ${loadedCard.name}`);
+      let loadedCard, loadedCardRef;
+      try {
+        loadedCard = await loadCard(fullRef, { loader: this.loader });
+        loadedCardRef = identifyCard(loadedCard);
+        if (!loadedCardRef) {
+          throw new Error(`could not identify card ${loadedCard.name}`);
```

Comment:
It's kind of awkward to throw an error and then immediately catch it and return an object with serializable error... but I suppose with this we can be sure that the error gets serialized with `serializableError`